### PR TITLE
Fix persistence archive option

### PR DESCRIPTION
**Pull request by wxmiked**
This resolves issue #363. It's a pretty minimal fix. Details of this fix can be found in [this comment](https://github.com/argoproj/argo-helm/issues/323#issuecomment-631158261) for that same issue.

Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo/values.yaml [hunk 1]
```diff
@@ -33,7 +33,8 @@ controller:
     enabled: false
     path: /metrics
     port: 8080
-  persistence: {}
+  persistence:
+    archive: false
```
Comment (stefansedich) [inline charts/argo/values.yaml:37]: Is the point here just to document this is an option? if so default is already false? perhaps we can just add it as a comment to document the option.

Comment (wxmiked) [reply charts/argo/values.yaml:37]: @stefansedich you're right. A commented out version of this would be just fine. I will make this change.

Comment (wxmiked) [reply charts/argo/values.yaml]: @stefansedich moved to a comment in the latest commit.

Comment (stefansedich) [reply charts/argo/values.yaml]: Thanks! mind just rebasing and bumping the chart version to the next available?

Comment (wxmiked) [reply charts/argo/values.yaml]: @stefansedich turns out the exact same issue was raised in this [accepted PR](https://github.com/argoproj/argo-helm/pull/363). So there's no reason to merge this. Thanks for reviewing this tho. I'm closing this MR.

Comment (stefansedich) [reply charts/argo/values.yaml]: Oops my bad I should have remembered this considering I merged that one :laughing: thanks again!

**Pull request closed without merging**